cypress/support/utils/packing_material.js

```diff
@@ -28,5 +44,15 @@ function applyPackingMaterial(packingMaterial) {
 
     cy.writeIntoStringField('Name', packingMaterial.name);
     cy.selectInListField('M_Product_ID', packingMaterial.product);
+
```

Comment:
shouldn't here be ` cy.writeIntoStringField('Length', packingMaterial.length);`, and the same to the others?

Reply:
done